plugins/che-plugin-analytics/che-plugin-analytics-wsagent/src/main/java/com/redhat/che/plugin/analytics/wsagent/AnalyticsEvent.java

```diff
@@ -19,7 +19,9 @@
   WORKSPACE_USED("Use Workspace in Che"),
   WORKSPACE_INACTIVE("Keep Workspace Inactive in Che"),
   WORKSPACE_STOPPED("Stop Workspace in Che"),
-  EDITOR_USED("Edit Workspace File In Che", 30, new String[] {PROGRAMMING_LANGUAGE});
+  EDITOR_USED("Edit Workspace File In Che", 30, new String[] {PROGRAMMING_LANGUAGE}),
+  PUSH_TO_REMOTE("Push to remote"),
+  COMMIT_LOCALLY("Commit locally");
```

Comment:
I would probably add an expected duration to both commit and push event.
Maybe something like 10 seconds ? That seems reasonable as the duration of such an action.

Adding an expected duration also avoids sending 2 events too close from one another in time, which, afaict, Woopra doesn't like much.

Reply:
Fixed